release_notes/Service_Fabric_ReleaseNotes_90CU5.md

```diff
@@ -0,0 +1,73 @@
+# Microsoft Azure Service Fabric 9.0 Cumulative Update 5.0 Release Notes
+
+This release will only be available through manual upgrades. Clusters set to automatic upgrades will not receive this release. For how to configure upgrades, please see [classic](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-upgrade) or [managed](https://docs.microsoft.com/en-us/azure/service-fabric/how-to-managed-cluster-configuration) documentation.
+
+## Contents
+
+* [Service Fabric Packages and Versions](#service-fabric-packages-and-versions)
+* [Service Fabric Feature and Bug Fixes](#service-fabric-feature-and-bug-fixes)
+* [Retirement and Deprecation Path Callouts](#retirement-and-deprecation-path-callouts)
+* [Repositories and Download Links](#repositories-and-download-links)
+
+
+## Service Fabric Packages and Versions
+
+The following packages and versions are part of this release:
+
+| Service | Platform | Version |
+|---------|----------|---------|
+|Service Fabric Runtime| Ubuntu 18 <br> Ubuntu 20 <br> Windows | 9.0.1148.1 <br> 9.0.1148.1 <br> 9.0.1155.9590 |
+|Service Fabric for Windows Server|Service Fabric Standalone Installer Package | 9.0.1155.9590 |
+|.NET SDK |Windows .NET SDK <br> Microsoft.ServiceFabric <br> Reliable Services and Reliable Actors <br> ASP.NET Core Service Fabric integration| 6.0.1155  <br> 9.0.1155 <br> 6.0.1155 <br> 6.0.1155 |
+|Java SDK  |Java for Linux SDK  | 1.0.6 |
+|Service Fabric PowerShell and CLI | AzureRM PowerShell Module  <br> SFCTL |  0.3.15  <br> 11.0.1 |
+
+
+## Service Fabric Feature and Bug Fixes
+
+| Versions | IssueType | Description | Resolution | 
+|-|-|-|-|
+| **Windows - 9.0.1155.9590<br>Ubuntu 18 - 9.0.1148.1<br>Ubuntu 20 - 9.0.1148.1** | **Bug** | Backup Restore Service(BRS) | **Brief Description**: OnDatalossAsync fails for NetCore applications due to System.MissingMethodException: Method is not found: 'Void System.Fabric.Common.Tracing.FabricEvents.BRSInfoPartitionEvent. This results in the Service partition getting stuck in reconfiguring in the case of data loss or QuorumLoss.<br>**Fix**: Rebuild SF application using SF SDK version 6.0.1107.9590 and upgrade/redeploy applications or Rollback cluster to 9.0.1028.9590 till we have a CU with fix for this regression
```

Comment:
When providing abbreviations in brackets, provide a space between the expansion and the brackets.
E.g.
Instead of "Backup Restore Service(BRS)" -- use -- "Backup Restore Service (BRS)"

Reply:
multiple such instances